timeline-chart/src/time-graph-container.ts

```diff
@@ -32,6 +32,16 @@ export class TimeGraphContainer {
         } else {
             canvas = extCanvas;
         }
+
+        let hasCanvas = true;
+
+        if (PIXI.utils.isWebGLSupported()) {
+            if (!canvas.getContext('webgl2')) {
+                hasCanvas = true;
+            } else {
+                hasCanvas = false;
+            }
+        }
```

Comment:
IMHO, this form and name makes it slightly more readable.

```suggestion
        let noWebgl2 = !PIXI.utils.isWebGLSupported() || !canvas.getContext('webgl2');
```

Reply:
Updated the commit with suggested changes.